### ISSUE TITLE
Update Stylelint to 4.2.0 to fix "rule-properties-order" error

### ISSUE THREAD

**Issue by danieldiekmeier** [AtomLinter/linter-stylelint#77]
See https://github.com/stylelint/stylelint/issues/665.

<img width="518" alt="screen shot 2016-02-05 at 21 14 41" src="https://cloud.githubusercontent.com/assets/2098462/12857976/7f365dd2-cc4d-11e5-89f4-461b13efc516.png">

This issue is fixed in 4.2.0, but still exists when using this plugin in Atom.

**Comment by Arcanemagus** [comment]
Just an FYI: `npm update` in the package directory would have fixed this for you, as we are marked as compatible with `^4.1.0`, but forcing an update doesn't hurt anything so why not :wink:.

**Comment by Arcanemagus** [comment]
Forced update pushed out in v1.10.4.

**Comment by ntwb** [comment]
@Arcanemagus You are correct, though, within Atom Editor itself, the version of stylelint included with linter-stylelint would remain as `4.2.0` as the Atom Editor doesn't do a `npm update` internally.

Before updating to v1.10.4:
```sh
$ cd .atom/.apm/stylelint

netweb@MacBook ~/.atom/.apm/stylelint
$ ls
total 0
drwxr-xr-x  11 netweb staff 374 Jan 28 22:58 .
drwxr-xr-x 747 netweb staff 25K Feb  3 17:34 ..
drwxr-xr-x   4 netweb staff 136 Sep 11 13:36 1.0.1
drwxr-xr-x   4 netweb staff 136 Sep 25 12:41 1.1.0
drwxr-xr-x   4 netweb staff 136 Oct 23 15:49 2.0.0
drwxr-xr-x   4 netweb staff 136 Nov 23 16:59 2.3.4
drwxr-xr-x   4 netweb staff 136 Dec 14 09:13 3.1.2
drwxr-xr-x   4 netweb staff 136 Dec 21 19:22 3.1.4
drwxr-xr-x   4 netweb staff 136 Jan 16 00:39 3.2.2
drwxr-xr-x   4 netweb staff 136 Jan 19 12:20 3.2.3
drwxr-xr-x   4 netweb staff 136 Jan 28 22:58 4.1.0
```
